### DISCUSSION TITLE
Model description changed

### DISCUSSION THREAD

**Discussion by Str1atum** [teslamate-org/teslamate#2036] [Q&A]
Yesterday in the dashboard my model description suddenly changed (see screenshot). I have Model 3 LR, nor sure what stood there before but now it's not correct for sure any more.

<img width="599" alt="Bildschirmfoto 2021-10-06 um 07 20 15 AM" src="https://user-images.githubusercontent.com/76776216/136144952-0b4a3681-e442-4540-be56-8d5257a20a86.png">

**Comment by init6-git** [comment]
It comes from the Trim Badging field. Not sure if this has changed recently or Teslamate is now displaying it. I don't recall seeing LR on the Teslamate screen though.  I thought it was just Model 3.

There are claims on Tesla Motors Club that this has been in this state for about 6 months on TelsaLogger.  Came here to check the release notes in case it's a new addition - will do that now.

**Comment by init6-git** [comment]
I note it has also been raised as an issue:

https://github.com/adriankumpf/teslamate/issues/2030
